functions/Enums.js

```diff
@@ -8,23 +8,32 @@
 /*global exports:true*/
 /*eslint no-undef: "error"*/
 
-exports.DateTimeFormat = {
-    DATE: 'YYYY-MM-DD',  // '2018-03-24'
-    TIME: 'HH:mm',  // '15:30' (24hr)
+// TODO: temporary for pilot, manually setting a fixed umbrella
+exports.UmbrellaId = {
+    TEST: 'PrE7oCZajNcRAOXNAFwRQizQQcJ3'
+    // TODO: create a real umbrella for UW
+};
+
+exports.StringFormat = {
+    DATE: 'MM/DD/YYYY', // '03/14/2018'
+    TIME: 'HH:mm', // '15:30' (24hr),
+    PHONE: '[0-9]{3}-[0-9]{3}-[0-9]{4}', // '206-345-7890',
+    WEEKDAY: 'dddd', // 'Wednesday'
+    WEEKDAY_WITH_DATE: 'dddd, MM/DD/YYYY' // 'Wednesday, 03/14/2018'
 };
 
 exports.AccountType = {
     UMBRELLA: 'umbrella',
     DONATING_AGENCY_MEMBER: 'donating_agency_member',
-    RECEIVING_AGENCY : 'receiving_agency',
+    RECEIVING_AGENCY: 'receiving_agency',
     DELIVERER_GROUP: 'deliverer_group'
 };
 
 // For now (as of 3/5/2018), the only umbrella
 // account type is school (eg UW). In the future,
 // there could be other types such as corporate.
 exports.UmbrellaType = {
-    SCHOOL: 'school',
+    SCHOOL: 'school'
```

Comment:
This is a separate `Enums.js` file that lives under `functions/` rather than `src/`. This is the whole thing about the Enums hack in #22 (where i couldn't figure out a way to share the Enums file between React and Cloud Functions). So the hack was we would only change Enums in `src/Enums.js` and copy the content over to `functions/Enums.js` using `python hacks/cp_enums.py`. So all the changes in this file is just syncing `functions/Enums.js` with `src/Enums.js` by running the script.

Reply:
Yea I think John was talking about a style thing:  __always leaving a comma after each line in object literals__, which I also agree. it's actually an intentionally designed new feature in many languages to make version control cleaner. New versions of js even support trailing commas in function parameters.

Reply:
Yeah i understood it was about the style, and I was trying to explain that these are changes in `src/Enums.js` and not actually changes I made here. If we want to change it back, I would need to change `src/Enums.js`. 
I do agree that keep trailing commas seems like a better style. We could add it to the `lint` rule later for it to be enforced :) (added a trello card)